servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -19,6 +19,29 @@
 nessie.server.default-branch=main
 nessie.server.send-stacktrace-to-client=false
 
+## Nessie authorization settings
+### This will perform authorization on branches/tags and contents where rule definitions are
+### using a Common Expression Language (CEL) expression (an intro to CEL can be found at https://github.com/google/cel-spec/blob/master/doc/intro.md).
+### Rule definitions are of the form nessie.server.authorization.rules.<ruleId>="<rule_expression>"
+### Available variables within the <rule_expression> are: 'op' / 'role' / 'ref' / 'path'
+### The 'op' variable in the <rule_expression> can be any of:
+### 'CREATE_REFERENCE', 'DELETE_REFERENCE', 'READ_ENTRIES', 'LIST_COMMIT_LOG', 'COMMIT_CHANGE_AGAINST_REFERENCE', 'ASSIGN_REFERENCE_TO_HASH', 'UPDATE_ENTITY', 'READ_ENTITY_VALUE', 'DELETE_ENTITY'
+### The 'role' refers to the user's role and can be any string
```

Comment:
I assume, it's actually okay to rename `role` to `user` in this case - `user` is also probably easier to grasp.

Reply:
for me `user` seems to be more narrow in describing what it is, therefore I thought keeping `role` gives us slightly more flexibility in describing what we deal with here